packages/typescript-typings/types/solidity-parser-antlr/index.d.ts

```diff
@@ -0,0 +1,808 @@
+// Type definitions for solidity-parser-antlr 0.3.2
+// Project: https://github.com/federicobond/solidity-parser-antlr
+// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>
+//                 Alex Browne <https://github.com/albrow>
+//                 Remco Bloemen <https://github.com/recmo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
```

Comment:
Add a comment why you copied it back to the repo. What is changed here. Also add a link to a DT PR.